src/main/java/core/basesyntax/service/impl/TransactionParserImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.TransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionParserImpl implements TransactionParser {
+    @Override
+    public List<String[]> parseInputDate(List<String> listOfOperations) {
+        List<String[]> result = new ArrayList<>();
+        for (int i = 1; i < listOfOperations.size(); i++) {
+            result.add(listOfOperations.get(i).split(","));
+        }
+        return result;
+    }
```

Comment:
<img width="814" alt="image" src="https://user-images.githubusercontent.com/81781479/215590660-e20be7c0-8247-4cef-b9a0-c57c09e5bd24.png">
not fixed - lets use Transaction class